get-started.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Getting Started
+summary: Quick list of instructions and hints to get started with NServiceBus and the Particular Service Platform
+reviewed: 2016-12-05
+---
+
+There is a variety of starting points for NServiceBus and the platform. The path taken will depend on an individual's specific experience and preferences for learning.
+
+
+## High Level Content
+
+These articles are recommended to gain an understanding of the key concepts and technologies used within the Platform.
+
+ * [Service Platform Overview](/platform/): Provides an overview of the various parts of Platform; [NServiceBus](/nservicebus/), [ServiceControl](/servicecontrol/), [ServiceInsight](/serviceinsight/) and [ServicePulse](/servicepulse/).
+ * [Architectural Principles](/nservicebus/architecture/principles.md)
+ * [Concepts Overview](/nservicebus/concept-overview.md): A high level overview of the concepts, features and vernacular of NServiceBus.
+ * [Hosting](/nservicebus/hosting/): Provides information on the various approaches to hosting an instance of NServiceBus.
+
+
+## Tutorials
+
+The [Messaging Basics Tutorial](/tutorials/nservicebus-101/) gives a guided walk though of building an NServiceBus system and introduces the basic concepts of messaging. The tutorial also has a downloadable and runnable solution to illustrate the finished product.
+
+
+## Go straight to runnable code
+
+Samples take a "download first" approach and then provided an explanation of how the given scenario, technology or concept.
```

Comment:
> Samples take a "download first" approach and then provided an explanation of how the given scenario, technology or concept.

Samples are fully functional solutions that can be run by hitting F5. The samples allow to learn by exploring code. Each includes an explanation of the scenario, technology or concept that it illustrates.